azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/ExpressRouteCircuit.java

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.network;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.apigeneration.Method;
+import com.microsoft.azure.management.network.implementation.ExpressRouteCircuitInner;
+import com.microsoft.azure.management.network.implementation.NetworkManager;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.GroupableResource;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.Resource;
+import com.microsoft.azure.management.resources.fluentcore.model.Appliable;
+import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
+import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
+import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
+
+import java.util.Map;
+
+/**
+ * Entry point for Express Route Circuit management API in Azure.
+ */
+@Fluent
+@Beta(Beta.SinceVersion.V1_4_0)
+public interface ExpressRouteCircuit extends
+        GroupableResource<NetworkManager, ExpressRouteCircuitInner>,
+        Refreshable<ExpressRouteCircuit>,
+        Updatable<ExpressRouteCircuit.Update> {
+
+    // Actions
+
+    /**
+     * @return entry point to manage express route peerings associated with express route circuit
+     */
+    @Method
+    ExpressRouteCircuitPeerings peerings();
+
+    // Getters
+
+    /**
+     * @return the SKU
+     */
+    ExpressRouteCircuitSku sku();
```

Comment:
I guess this is auto-gened class, which contains `tier` and `family`. Usually we create an enum for such skus.. For example an enum for this type can have the values

```java
ExpressRouteCircuitSkuTypes.STANDARD_UNLIMITEDDATA
ExpressRouteCircuitSkuTypes.STANDARD_METEREDDATA
ExpressRouteCircuitSkuTypes.PREMIUM_UNLIMITEDDATA
ExpressRouteCircuitSkuTypes.PREMIUM_METEREDDATA
```

where _ separates skuTier and family.